code/modules/cooking/machinery/cooking_machines/_cooker_output.dm

```diff
@@ -22,37 +22,51 @@
 		create_reagents(size*8 + 10)
 	update_icon()
 
-/obj/item/reagent_containers/food/snacks/variable/update_icon()
+/obj/item/reagent_containers/food/snacks/variable/on_reagent_change()
+	return
+
+/obj/item/reagent_containers/food/snacks/variable/proc/update_prefix()
+	switch(scale)
+		if (0 to 0.8)
+			prefix = "small"
+		if (0.8 to 1.2)
```

Comment:
there's no intermediate/medium size? 

should remove extra large and just bump everything up

Reply:
Bugfix PR.